Tests/phpunit/bootstrap.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+declare(strict_types=1);
+
+require __DIR__ . '/../../vendor/autoload.php';
```

Comment:
just curious why not put `vendor/autoload.php` directly into the `phpunit.xml.dist` then we won't need this file I think.